core/check/ifZero_test.go

```diff
@@ -0,0 +1,32 @@
+package check_test
+
+import (
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go/core/check"
+	"github.com/stretchr/testify/assert"
+)
+
+type testStruct struct {
+	a uint64
+	b uint32
+	c uint
+	d string
+}
+
+func TestCheckForZeroUintFields(t *testing.T) {
+	zero := testStruct{}
+
+	err := check.ForZeroUintFields(zero)
```

Comment:
This will return an error on the "a" value. The other uint32 and uint values will not be checked

Reply:
done.

Reply:
ok